### PR TITLE
Load ses mehrdad

### PR DESCRIPTION
**Pull request by mehrdadhalali**

### CODE REVIEW

File: load_to_rds/create_html_message.py [hunk 1]
```diff
@@ -30,7 +29,9 @@ def format_price(price: int) -> str:
 
 def create_html(games: list[dict], genre: str) -> str:
     """Creates an HTML of the games of a specific genre."""
-    message = put_in_tag(f"Here is all of the newly released games of the {format_genre_text(genre)} genre:",
+
+    genre_formatted = format_genre_text(genre)
+    message = put_in_tag(f"Here is all of the newly released games of the {genre_formatted} genre:",
```
Comment (paulauus) [inline load_to_rds/create_html_message.py:34]: Inglish. The games are plural, surely. I'm Mehrdading this grammar!

File: load_to_rds/email_subscribers.py [hunk 2]
```diff
@@ -40,15 +39,13 @@ def send_genre_emails(scraped_data: list[dict]):
...
-            print(genre)
             genre_games = get_games_by_genre(genre, scraped_data)
             to_send = create_html(genre_games, genre)
 
             subscriber_emails = item["subscribers"]
-            print(subscriber_emails)
 
             ses.send_email(
-                Source="trainee.mehrdad.halali@sigmalabs.co.uk",
+                Source=ENV["SENDER_EMAIL_ADDRESS"],
```
Comment (paulauus) [inline load_to_rds/email_subscribers.py:48]: Good idea to not get spammed.

**Review by paulauus — DISMISSED** [review]

**Review by MichaelHassan — DISMISSED** [review]
I like the changes, now the emails look more professional and less like spam.

**Review by Leon-JS02 — DISMISSED** [review]

**Review by Leon-JS02 — APPROVED** [review]
Reapproval. Liking the changes.

**Review by paulauus — APPROVED** [review]

**Pull request merged**